prometheus_client/gc_collector.py

```diff
@@ -3,13 +3,17 @@
 from __future__ import unicode_literals
 
 import gc
+import os
 import time
 
 from . import core
 
 class GCCollector(object):
     """Collector for Garbage collection statistics."""
     def __init__(self, registry=core.REGISTRY, gc=gc):
+        if 'prometheus_multiproc_dir' in os.environ:
```

Comment:
Can you add a comment with a link to the issue? We can revert this once it's fixed.